python-port/setup.py

```diff
@@ -0,0 +1,39 @@
+import setuptools
+
+
+def readme():
+    with open("README.rst") as fh:
+        return fh.read()
+
+setuptools.setup(name="sendpy",
+      version="1.0",
+      description="Email and text notification program",
+      long_description=readme(),
+      long_description_content_type="text/markdown",
+      url="https://github.com/tdulcet/Send-Msg-CLI",
+      author="Daniel Connelly and Teal Dulcet",
+      author_email="connellyd2050@gmail.com",
+      license="GPL",
+      packages=["sendpy"],
+      scripts=["bin/sendpy"],
+      classifiers=[
+          "Programming Language :: Python :: 3.6",
+          "Programming Language :: Python :: 3.7",
+          "Programming Language :: Python :: 3.8",
+          "Programming Language :: Python :: 3.9",
+          "Programming Language :: Python :: 3.10",
+          "Programming Language :: Python :: 3.11",
+          "Programming Language :: Python :: 3.12",
+          "License :: OSI Approved :: GNU General Public License (GPL)",
+          "Operating System :: OS Independent",
+      ],
+      keywords="sendpy email e-mail send text notification g-mail gmail",
```

Comment:
Left over gmail tag in here I think, my mistake.

Reply:
OK, I removed it. Note SendPy of course still fully supports Gmail.